AJOAuth2/ViewControllers/LeftViewController.m

```diff
@@ -60,6 +62,12 @@ - (void)viewWillAppear:(BOOL)animated {
         
         // Email Address
         _emailLabel.text = user.emailAddress;
+        
+        // ImageView
+        if (user.avatarImageURLString.length > 0)
+            _avatarImageView.image = [UIImage imageWithData:[Helper avatarImageUrl:user.avatarImageURLString]];
```

Comment:
don;t you need image data as PNG etc?

And are you caching it or not? i.e. unless url is diff you should pic old one